src/Symfony/Component/Form/Test/TypeTestCase.php

```diff
@@ -34,6 +35,22 @@ protected function setUp()
         $this->builder = new FormBuilder(null, null, $this->dispatcher, $this->factory);
     }
 
+    protected function getExtensions()
```

Comment:
Changes pushed. It's just a POC, so I just want to get feedback on the idea. If everyone is happy with the approach then I'll finish the implementation